### PR TITLE
[Hub Generated] Review request for Microsoft.Sql to add WorkloadGroups and WorkloadClasifiers to API 2019-06-01-preview

### PR DESCRIPTION
**Pull request by H-Huang**
### Workload Management

Adding new WorkloadGroups.json and WorkloadClassifiers.json swagger files and their examples. These are also part of a new API version 2019-06-01-preview.

Workload groups and workload classifiers are new objects in Azure SQL Data Warehouse which allow customer to manage their instance resources and workloads.

The new swagger files contain the CRUD operations which include
- List the workload groups / workload classifiers
- Get the workload groups / workload classifiers
- Create or update the workload groups / workload classifiers
- Delete the workload groups / workload classifiers

If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/H-Huang/azure-rest-api-specs/dev-sql-Microsoft.Sql-2019-06-01-preview...master/).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by yaohaizh** [comment]
@H-Huang some examples are ill-formated. https://dev.azure.com/azure-sdk/public/_build/results?buildId=119412&view=logs&jobId=8c9a658c-6596-5897-87e8-abb327497d15

**Comment by yaohaizh** [comment]
@jaredmoo please help review this PR as the code owner. Thanks for your help a lot.

File: specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/DataWarehouseWorkloadClassifiers.json [hunk 1]
```diff
@@ -0,0 +1,484 @@
...
+          },
+          "202": {
```
Comment (ryansbenson) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/DataWarehouseWorkloadClassifiers.json]: Resource creation shouldn't return 202:
https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#creating-or-updating-resources

File: specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/DataWarehouseWorkloadGroups.json [hunk 2]
```diff
@@ -0,0 +1,463 @@
...
+          },
+          "202": {
```
Comment (ryansbenson) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/DataWarehouseWorkloadGroups.json]: Resource creation shouldn't return 202:
https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#creating-or-updating-resources

File: specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/examples/GetWorkloadClassifier.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      "body": {
+        "properties": {
+          "memberName": "dbo",
+          "label": "test_label",
+          "context": "test_context",
+          "startTime": "12:00",
+          "endTime": "14:00",
+          "importance": "high"
+        },
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/Default-SQL-SouthEastAsia/providers/Microsoft.Sql/servers/testsvr/databases/testdb/workloadGroups/smallrc",
```
Comment (ryansbenson) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/examples/GetWorkloadClassifier.json]: Across the samples, the classifier resource ids and types should not be for: Microsoft.Sql/servers/databases/dataWarehouseWorkloadGroups

File: specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/DataWarehouseWorkloadClassifiers.json (hunk 1 above)
Comment (jamestao) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/DataWarehouseWorkloadClassifiers.json]: 202 is allowed for resource creation, see #6 of:
https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#202-accepted-and-location-headers

File: specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/examples/GetWorkloadClassifier.json (hunk 3 above)
Comment (H-Huang) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/examples/GetWorkloadClassifier.json]: Thanks! That was a good catch. Addressed in latest commits

File: specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/DataWarehouseWorkloadClassifiers.json (hunk 1 above)
Comment (H-Huang) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/DataWarehouseWorkloadClassifiers.json]: I followed existing examples which have the similar pattern of 202 for creation [https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/managedDatabases.json](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/managedDatabases.json)

Comment (ryansbenson) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/DataWarehouseWorkloadClassifiers.json]: I forgot about this caveat, its not normally used.

File: specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/examples/CreateOrUpdateWorkloadClassifierMax.json [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "memberName": "dbo",
+          "label": "test_label",
+          "context": "test_context",
+          "startTime": "12:00",
```
Comment (jaredmoo) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/examples/CreateOrUpdateWorkloadClassifierMax.json]: Swagger & ARM team reviewers, what is the guidance for exposing "time of day" (i.e. just time, not date+time)?

**Review by jaredmoo — APPROVED** [review]

File: specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/examples/CreateOrUpdateWorkloadClassifierMax.json (hunk 4 above)
Comment (yaohaizh) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/examples/CreateOrUpdateWorkloadClassifierMax.json]: > Swagger & ARM team reviewers, what is the guidance for exposing "time of day" (i.e. just time, not date+time)?
@ryansbenson, what is the suggestion here?

**Review by yaohaizh — APPROVED** [review]

**Pull request merged**